content/Classes/Classes.facts.md

```diff
@@ -0,0 +1,21 @@
+### Facts for Classes lesson:
+
+A class is a blueprint that defines the characteristics and behaviors of all objects of a specific kind. It has properties and methods as its basic building blocks.
+
+A property is a variable that is defined inside a class. 
+
+A method is a function inside a class that performs a specific action.
+
+To declare a class, use the `class` keyword followed by the class name, and a pair of curly braces `{}` that enclose the definition of the `properties` and `methods` of a class. 
+
+The following is an example of a class declaration:
+
+```php
+class MyClass {
+    // properties and methods here...
+}
+```
+
+A class name may start with a letter or an underscore `_`, followed by any combination of alphanumeric characters and underscores (`A-Z`, `a-z`, `0-9`, `_`). 
+
+It is good practice to capitalize the first letter of a class name to easily identify them from other variables. Some examples of good class names are `MyClass`, `SuperHero`, and `Car`.
```

Comment:
It should be capitalized as it is the common convention in the public and it would be much easier for others to understand the code. And also, most editors will not highlight uncapitalize class names which they might think the name is wrong.